Sources/TuistKit/Commands/EnvKey/EnvKey.swift

```diff
@@ -224,6 +224,10 @@ public enum EnvKey: String, CaseIterable {
     // SESSION
     case whoamiPath = "TUIST_WHOAMI_PATH"
 
+    // CHANGE USERNAME
+    case changeUsernameName = "TUIST_CHANGE_USERNAME_NAME"
+    case changeUsernamePath = "TUIST_CHANGE_USERNAME_PATH"
```

Comment:
I'd also add the super command key in the value, in this case it would be `TUIST_ACCOUNT_CHANGE_USERNAME_NAME`.

We could (and probably should) automatically synthesize these at some point